thorlcr/thorutil/thbuf.cpp

```diff
@@ -2549,6 +2538,7 @@ class CSharedFullSpillingWriteAhead : public CInterfaceOf<ISharedRowStreamReader
         outputStream->flush();
         totalInputRowsRead.fetch_add(newRowsWritten);
         tempFileOwner->noteSize(iFileIO->getStatistic(StSizeDiskWrite));
+        ::mergeStats(inactiveStats, iFileIO);
```

Comment:
I think this is going to double count the stats.  Each time it is called it will add the stats from the previous call as well as the rows just written.